overpower.py

```diff
@@ -141,11 +141,12 @@ def _handle_ps1_profiler_output(self, output: Dict[str, Any], result: Result, fi
             # If a signature was raised in a previous layer profile, then don't report it
             if tag in previous_signatures:
                 continue
-            profiler_sig_section = ResultSection(
+            profiler_sig_section = ResultTextSection(
                 title_text=f"Signature: {tag}",
                 parent=suspicious_res_sec,
-                body=f"Marks: {', '.join(details['marks'])}" if details['marks'] else None
             )
+            if details['marks']:
```

Comment:
is 'marks' always given? If not, maybe a `details.get('marks')` could save us from another PR?